README.md

```diff
@@ -291,6 +291,57 @@ sum by (thread_name) (idelta(tnt_cpu_thread[$__interval]))
   / scalar(idelta(tnt_cpu_total[$__interval]) / tnt_cpu_count)
 ```
 
+## Cartridge role
+
+`cartridge.roles.metrics` is a role for [tarantool/cartridge](https://github.com/tarantool/cartridge). It allows to use default metrics in a cartridge application and manage them with config.
+
+### Usage
+
+1. Add `metrics` package to dependencies in the .rockspec file. Make sure that you are using version 0.3.0 or higher.
+```lua
+dependencies = {
+    ...
+    'metrics == 0.3.0-1',
```

Comment:
Few lines before you say "0.3.0 or higher" I think it could be changed to `'metrics >= 0.3.0-1'` to be a bit more consistent.